.github/workflows/android-ci.yml

```diff
@@ -13,16 +13,16 @@ jobs:
     runs-on: ${{ matrix.os }}
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
       - name: Set up Node.js
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v4
         with:
           node-version: 20
           cache: 'yarn'
       - name: Set up JDK
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v4
         with:
-          distribution: temurin
+          distribution: zulu
           java-version: 11
```

Comment:
best we stick to [zulu@11 for RN 0.72](https://reactnative.dev/docs/0.72/environment-setup)
then introduce [JDK 17 in RN 0.73](https://reactnative.dev/docs/0.73/environment-setup)
like [upstream](https://github.com/react-native-webview/react-native-webview/pull/3347/files#diff-61285a9c4b09f2f5fe6d2ebd01e2ee1778a3412e5129cbbe69756d71b0774b84)